### PR TITLE
Add behat tests.

### PR DESCRIPTION
**Pull request by YuriGizra**
#1

### CODE REVIEW

**Comment by YuriGizra** [comment]
@amitaibu, Ready for review.

File: behat/features/bootstrap/FeatureContext.php [hunk 1]
```diff
@@ -23,6 +23,86 @@ public function iVisitTheHomepage() {
   }
 
   /**
+   * @When I fill require fields and add to cart
+   */
+  public function iFillRequireFieldsAndAddToCart()
+  {
```
Comment (amitaibu) [inline behat/features/bootstrap/FeatureContext.php]: move up

File: behat/features/cart.feature [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Feature: Add to cart
+  In order to be able to add to cart.
+  As an anonymous user
+  We need to be able to add product to cart
+
+  @javascript
+  Scenario: Visit product page
+    Given I am an anonymous user
+    When  I visit "Product.aspx?p=3393&m=390"
+    And   I fill require fields and add to cart
```
Comment (amitaibu) [inline behat/features/cart.feature]: `And I add initials and add to cart`

**Comment by YuriGizra** [comment]
@amitaibu, fixed.

**Pull request merged**